src/middlewared/middlewared/plugins/replication.py

```diff
@@ -921,13 +629,7 @@ def new_snapshot_name(self, naming_schema):
         return datetime.now().strftime(naming_schema)
 
     # Legacy pair support
-    @private
-    @accepts(Dict(
-        "replication-pair-data",
-        Str("hostname", required=True),
-        Str("public-key", required=True),
-        Str("user", null=True),
-    ))
+    @api_method(ReplicationPairArgs, ReplicationPairResult, private=True)
```

Comment:
For private methods like this, I think it's better to remove `api_method` overhead altogether. It's not like the arguments to this function are complicated in how they need to be validated.

Reply:
Can I still use them though?

Reply:
Yeah you can but I don't see the benefit because of all the overhead it adds. But yeah, you definitely still can and I'm not asking for you to change it :+1: . Just wanted to mention it here.